### ISSUE TITLE
Bloc de texte administrable depuis le BO PrestaShop avec le module adéquat

### ISSUE THREAD

**Issue by Axecibles-Dev** [GuillaumeFaidherbe/Axeci-Boutik#2]
Pour la gestion du bloc de texte administrable depuis le BO PrestaShop, il existe un module natif plus adéquat que le module de bloc CMS destiné à afficher des informations sur tout le site.

**Comment by GuillaumeFaidherbe** [comment]
Bonjour,

Effectivement, nous pouvons créer un nouveau bloc modifiable via le BO dans
préférence -> CMS, puis nous pouvons l'afficher grâce au module Bloc CMS où
nous devons créer un nouveau bloc ( ici test) et mettre ce que l'on veut
qu'il affiche comme block CMS créer dans Préférences -> CMS (ci-joint les
screens qui montre le bloc)

Merci,
Guillaume Faidherbe

Le mer. 2 déc. 2020 à 10:49, Axecibles Dev <notifications@github.com> a
écrit :

**Comment by Axecibles-Dev** [comment]
Ou utiliser le module natif "Home text editor" :)

**Comment by GuillaumeFaidherbe** [comment]
Effectivement, après avoir installé ce module, il est en effet possible de
créer un bloc à l'accueil modifiable via le BO, merci

Le mer. 2 déc. 2020 à 12:25, Axecibles Dev <notifications@github.com> a
écrit :

**Comment by Axecibles-Dev** [comment]
Peux-tu mettre à jour la BDD sur le dépôt que je puisse vérifier ?

**Comment by GuillaumeFaidherbe** [comment]
C'est fait, merci

Le mer. 2 déc. 2020 à 12:52, Axecibles Dev <notifications@github.com> a
écrit :
